image_load.py

```diff
@@ -0,0 +1,75 @@
+import logging
+import random
+
+import requests
+
+from custom_errors import (
+    APINotAvailable,
+    PexelAPIKeyError,
+    JsonError
+)
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger("Image_Load_Logger")
+
+
+def search_photos(pexel_api_key, query="ocean", orientation="", size="", color="", page=1, per_page=15):
+    term = "search"
+    query = {"query": query, "orientation": orientation, "size": size,
+             "color": color, "page": page, "per_page": per_page}
+    photos = get_image_url(pexel_api_key, term, query, )
+    return photos
+
+
+def curated_photos(pexel_api_key, page=1, per_page=80):
+    term = "curated"
+    query = {"page": page, "per_page": per_page}
+    image_url = get_image_url(pexel_api_key, term, query)
+    return image_url
+
+
+def get_image_url(pexel_api_key, term, query):
+    api_url = "https://api.pexels.com/v1/"
+    headers = {"Authorization": pexel_api_key}
+    logger.info("Connecting to Pexels API...")
+    try:
+        response = requests.get(f"{api_url}{term}",
+                                headers=headers, params=query)
+        response.raise_for_status()
+    except requests.exceptions.HTTPError as e:
+        if response.status_code == 401:
+            logger.info("Problems with your Pexel_api_key")

```

Comment:
Actually, too much logging - makes it almost impossible to understand what's going on. Try to reduce the number of calls